src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Remote/IRemoteAutoInsertService.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor;
+using Microsoft.CodeAnalysis.Text;
+
+using Response = Microsoft.CodeAnalysis.Razor.Remote.RemoteResponse<Microsoft.CodeAnalysis.Razor.Protocol.AutoInsert.RemoteInsertTextEdit?>;
+
+internal interface IRemoteAutoInsertService : IDisposable
+{
+    ValueTask<Response> TryResolveInsertionAsync(
```

Comment:
:+1:

Reply:
It always returns a result, but it doesn't always resolve the edit, so Try seems appropriate to me, but I don't have strong feeling here. I made it Try since existing methods for auto insert edit resolve were Try. I'll rename it to Get.

InsertTextEdit should've been deleted per your suggestion, my bad. I eliminated all usages but forgot to actually delete the type.

Happy to rename to RemoteAutoInsertTextEdit to make it more specific, that's always good.